Makefile

```diff
@@ -199,15 +199,15 @@ $(GIT_SAMPLE_DATA_REPO_PATH): | $(DATA_DIR)
 	git -C $(GIT_SAMPLE_DATA_REPO_PATH) lfs install
 	git -C $(GIT_SAMPLE_DATA_REPO_PATH) checkout $(GIT_SAMPLE_DATA_REPO_REV)
 	git -C $(GIT_SAMPLE_DATA_REPO_PATH) lfs fetch
-	git -C $(GIT_SAMPLE_DATA_REPO_PATH) lfs checkout
+	git -C $(GIT_SAMPLE_DATA_REPO_PATH) lfs checkout $(GIT_SAMPLE_DATA_REPO_REV)
 
 $(GIT_TEST_DATA_REPO_PATH): | $(DATA_DIR)
 	-git clone $(GIT_TEST_DATA_REPO) $(GIT_TEST_DATA_REPO_PATH)
 	git -C $(GIT_TEST_DATA_REPO_PATH) fetch
 	git -C $(GIT_TEST_DATA_REPO_PATH) lfs install
 	git -C $(GIT_TEST_DATA_REPO_PATH) checkout $(GIT_TEST_DATA_REPO_REV)
 	git -C $(GIT_TEST_DATA_REPO_PATH) lfs fetch
-	git -C $(GIT_TEST_DATA_REPO_PATH) lfs checkout
+	git -C $(GIT_TEST_DATA_REPO_PATH) lfs checkout $(GIT_TEST_DATA_REPO_REV)
```

Comment:
Is that last `checkout` step needed? We've done the `lfs fetch` and should already be in `$(GIT_TEST_DATA_REPO_REV)` right?

Reply:
Yep, it should be fine without this change!  Removed in 89d236b1f